components/dashboard/Dashboard.js

```diff
@@ -175,18 +176,30 @@ export default function Dashboard(props) {
     allActionDispositionEvents,
     allTaskDispositionEvents,
     completedTasks,
+    confidentActivityLogEntries,
     historyLimit,
     recentActivityLogEntries,
+    activityLogEntriesCount,
     ...restProps
   } = props;
 
   const todoTaskLimit = 3;
 
+  const [activeDialog, setActiveDialog] = useState();
   const allApplicableTasks = tasksToShow(props);
   const doneTaskCount = allTaskDispositionEvents.length;
   const todoTaskCount = Math.min(allApplicableTasks.length - doneTaskCount, todoTaskLimit);
   const tasks = allApplicableTasks.slice(0, todoTaskCount + doneTaskCount);
-  const [activeDialog, setActiveDialog] = useState();
+  const totalActivitiesCount = useRef(activityLogEntriesCount);
```

Comment:
Here's where we could use some eyes, @rossdakin .
We only want to update the percentage passed to the Gauge component when the activities entries count has been updated. This happens after an entry is added to the activity entries log, resulting in the number of confident activity entries being updated before the total count was updated (so the gauge would flicker to the wrong value before correcting itself). The useRef is one approach, but totally open to others

Reply:
We can disregard this for now, in the next ticket (the mapping of the activities to categories) we have to retrieve the entire collection of activity entries. This will be moot.